config/initializers/redis.rb

```diff
@@ -0,0 +1,3 @@
+def redis
```

Comment:
Yeah, I can see the benefit of using the existing `Upaya` module. Although I must say that we do not ref that name very often in the context of this app, so it's actually kind of confusing to see it. `IdentityIdp` would be a lot easier to grok. 

For this reason, I am leaning toward having a class for this.  Probably not a service class since I usually use those to *do* something....but idk. Thinking...